src/os/linux/vm/os_perf_linux.cpp

```diff
@@ -911,14 +911,9 @@ int SystemProcessInterface::SystemProcesses::ProcessIterator::next_process() {
   }
 
   do {
-      entry = os::readdir(_dir);
```

Comment:
The declaration of entry should be removed.

Reply:
fixed